conans/client/packager.py

```diff
@@ -30,8 +30,8 @@ def new_method(pattern, src=""):
     conanfile.copy_bins = wrap(DEFAULT_BIN)
     conanfile.copy_res = wrap(DEFAULT_RES)
     try:
+        conanfile.package_folder = package_folder
```

Comment:
You already have the conanfile_directory() which is pointing to the build folder. I didnt want to duplicate that information.